exercises/sum-of-multiples/canonical-data.json

```diff
@@ -38,6 +38,15 @@
       },
       "expected": 23
     },
+    {
+      "description": "multiples of 3 or 5 up to 20",
```

Comment:
It would be better if the description would say why this case is important to test; that will help both students of Exercism who might want to understand why their implementation does not pass this test.

Here, you want to test the case where a single number is a multiple of two of the given factors. That should be made clear in the description.

The https://github.com/exercism/problem-specifications#test-data-format-canonical-datajson supports this by stating "description should [...] explain [...] why each case is there. For example, what kinds of implementation mistakes might this case help us find?"

Reply:
EBWODP: Incomplete sentence.

> help both students of Exercism who might want to understand why their implementation does not pass this test

"Both" implies two element, but in the sentence Iw rote, I provided one element.

Should add: "and track maintainers who are evaluating whether to keep this test or not." to serve as the second element.